hardware/setup/create_framework.tcl

```diff
@@ -45,32 +45,12 @@ if { [info exists ::env(HLS_SUPPORT)] == 1 } {
   set hls_support "not defined"
 }
 
-if { [info exists ::env(USE_PRFLOW)] == 1 } {
-  set use_prflow [string toupper $::env(USE_PRFLOW)]
-} else {
-  set use_prflow "FALSE"
-}
-
-if { [info exists ::env(CLOUD_USER_FLOW)] == 1 } {
-  set cloud_user_flow [string toupper $::env(CLOUD_USER_FLOW)]
-} else {
-  set cloud_user_flow "FALSE"
-}
-
 if { [info exists ::env(PSL_DCP)] == 1 } {
-  if { $cloud_user_flow == "TRUE" } {
-    set psl_dcp "FALSE"
-  } else {
-    set psl_dcp $::env(PSL_DCP)
-  }
+  set psl_dcp $::env(PSL_DCP)
 } else {
   set psl_dcp "FALSE"
 }
 
-if { ($use_prflow == "TRUE") && ($hls_support == "TRUE") } {
-  set action_dir $::env(ACTION_ROOT)/hw/vhdl
-}
-
```

Comment:
I am not sure if that really can be removed

Reply:
Due to your comment, I re tested the case KU3 + hls_helloworld on cloud build / Nimbix by doing **snap_config + make model + make image**
I guess the missing **action_dir** would have been seen during **make framework** of **make model** when getting the code generated by HLS but it worked ok through the whole flow.

Reply:
Configured N250S + hls_memcopy + cloud_user="FALSE"
Calling `make cloud_base` then leads to the following error:
```
ERROR: [Vivado 12-4865] Failed to create reconfig module as Module with an instance in active top cannot be made into reconfigurable module
make[1]: *** [cloud_base] Error 1
```

Reply:
Please let us focus on the ADKU3 card, to enable it on cloud as soon as possible. The regression tests on ADKU3 with CLOUD_USER_FLOW="FALSE" for the hls_memcopy action looks good